ompi/attribute/attribute.c

```diff
@@ -50,6 +52,8 @@
  *   means writing a pointer to an instance of something; changing the
  *   value of that instance will make it visible to anyone who reads
  *   that attribute value).
+ * - C also internally store some int attributes of a MPI_Win by value,
```

Comment:
"... store**s** ..."

Should probably also list MPI-3.1 Section 11.2.6 (pp414-415) to the list of sections to reference (earlier in this comment).